.github/workflows/benchmark_asterinas.yml

```diff
@@ -102,23 +102,42 @@ jobs:
         fi
         echo "ALERT_TOOL=$ALERT_TOOL" >> $GITHUB_ENV
         
+        TITLE=$(jq -r '.title' test/benchmark/${{ matrix.benchmark }}/config.json)
+        if [ -z "$TITLE" ]; then
+          TITLE=""
+        fi
+        echo "TITLE=$TITLE" >> $GITHUB_ENV
+        
         DESCRIPTION=$(jq -r '.description' test/benchmark/${{ matrix.benchmark }}/config.json)
         if [ -z "$DESCRIPTION" ]; then
           DESCRIPTION=""
         fi
         echo "DESCRIPTION=$DESCRIPTION" >> $GITHUB_ENV
 
+        DISPLAY=$(jq -r '.display' test/benchmark/${{ matrix.benchmark }}/config.json)
+        if [ "${DISPLAY}" = "true" ]; then
+          DISPLAY=true
+        else
+          DISPLAY=false
+        fi
+        echo "DISPLAY=$DISPLAY" >> $GITHUB_ENV
+
+        BENCHMARK_NAME=$(basename "${{ matrix.benchmark }}")
+        echo "BENCHMARK_NAME=$BENCHMARK_NAME" >> $GITHUB_ENV
+
     - name: Store benchmark results
-      uses: asterinas/github-action-benchmark@v2
+      uses: asterinas/github-action-benchmark@v3
```

Comment:
Ideally, the order of benchmark displayed in the summary table should be controlled by our script. Not this Github action. I guess that you have hardcoded some ordering rules in the action. This is not good. We will add more benchmarks in the future. You can't and shouldn't hardcode the ordering benchmark of every benchmark suite in the Github action.

Reply:
Then we should have an extra field to store the ordering rules. Now the ordering rule is hardcoded, which groups the benchmarks with the same prefix together. If we want to define the ordering, should we just follow the matrix of the workflow or define an ordering list?

Reply:
What is your proposal?

Reply:
Pre-defining an ordering list brings an extra step of operating the benchmark repository.
I prefer to apply the ordering of our benchmark workflow to the benchmark repository through our action. Ideally, we should finish the ordering when we update the benchmark data. So we just need to draw the figures which are already ordered. I will add such a feature to the benchmark action later.